content/messages/messages-fr.yaml

```diff
@@ -0,0 +1,120 @@
+---
+switch_to: "Passez en"
+hi_there: "Bienvenue chez nous !"
+choose_solutions: "Commencez par choisir l'une de nos solutions"
+the_pim: "Le PIM"
+best_container: "Le meilleur des conteneurs pour vos précieuses données produit"
+onboarder: "L'Onboarder"
+onboarder_description: "Récoltez facilement l'information produit de vos fournisseurs"
+
+
+older_versions: "Anciennes versions"
+can_we_help_you: "Que recherchez-vous ?"
+type_your_search: "Saisissez votre recherche ici..."
+search_example: "Par exemple, essayez"
+what_family: "Qu'est-ce qu'une famille ?"
+or: "ou"
+projects: "Projets"
+how_permissions: "Comment paramétrer les permissions ?"
+help_center_in_your_language: "Le help center est maintenant en français !"
+help_center_in_your_language_description: "Mais pas trop quand même... Et oui ! Il reste des articles à traduire. Comme toujours, nous, on croit à la puissance de l'open source. Du coup, si vous êtes arrivés au bout de vos séries Netflix et que vous avez du temps à tuer, n'hésitez pas à nous donner un coup d'main !"
+its_here: "On vous explique tout comment faire ici."
```

Comment:
hum, je ne vois pas trop le contexte mais "tout comment faire" me parait bizarre
peut-être "On vous explique comment faire ici." ?

Reply:
It was to make a bit of humor, but I guess it does not work. ;) I'll change.